ChatAPI.Domain/Entities/User.cs

```diff
@@ -14,7 +14,24 @@ public class User(string name, string phone)
 
 		[Required]
 		public DateTime CreatedOn { get; set; } = DateTime.UtcNow;
+		public IEnumerable<Friendship> Friendships
+		{
+			get
+			{
+				List<Friendship> friendships = [];
+				
+				if (SentFriendships is not null)
+					friendships.AddRange(SentFriendships);
+
+				if (ReceivedFriendships is not null)
+					friendships.AddRange(ReceivedFriendships);
+
+				return friendships;
+			}
+		}
 
 		public virtual ICollection<UserDevice>? UserDevices { get; set; }
-    };
+		public virtual ICollection<Friendship>? SentFriendships { get; set; }
+		public virtual ICollection<Friendship>? ReceivedFriendships { get; set; }
+	};
```

Comment:
Have you tested this? I do not think this will work the way you expect it to. I would like to talk this one over.

Reply:
lets discuss, but not block the PR because of it, do we have a deal? =)

Reply:
I wrote why I rejected the PR very clearly. It was not because of this.